src/crepe/facade/SDLContext.cpp

```diff
@@ -103,12 +105,24 @@ void SDLContext::handle_events(bool & running) {
 void SDLContext::clear_screen() { SDL_RenderClear(this->game_renderer.get()); }
 void SDLContext::present_screen() { SDL_RenderPresent(this->game_renderer.get()); }
 
+void SDLContext::set_rbg_texture(const std::shared_ptr<Texture>& texture, const uint8_t& r, const uint8_t& g, const uint8_t& b){
```

Comment:
willen we de shared_ptr nu al aanpassen of doen we dat later?

Reply:
hier gaan we het waarschijnlijk morgen (19-11-2024) over hebben

Reply:
Dit wordt opgelost als we een beter idee hebben over de assets.